web/src/contexts/LinksContextProvider.test.js

```diff
@@ -103,136 +103,78 @@ describe('LinksContextProvider', () => {
       activeSection,
       activities
     );
-    expect(testLinks.length).toBe(9);
+    expect(testLinks.length).toMatchSnapshot();
```

Comment:
```suggestion
    expect(testLinks).toMatchSnapshot();
```

One little typo here!